### ISSUE TITLE
Decide which couchdb package we use for jessie

### ISSUE THREAD

**Issue by varac** [pixelated/pixelated-platform#130]
There is only a couchdb 1.6 packaged for wheezy in the LEAP deb repo. 

I see 3 possibilities:

- use couchdb 1.4 from debian unstable (not a longterm solution because we woulnd't recieve security updates)
- try do do an official debian backport of couch 1.4 from unstable to jessie-backports
- package couch 1.6 ourselves for jessie

**Comment by cz8s** [comment]
I built couch 1.6 from the wheezy sources for jessie.
That package works and is installable on jessie.
But development for couch is definately focused on 
couch 2.0.
Using 1.6 for jessie works for now, but building is manual and we don't get any
security fixes. Maybe we should drop couch?

**Comment by varac** [comment]
from #couchdb irc:

```
varac_ hej all, i'm wondering if couchdb does security fixes for the latest release (1.6.1), and how i would get them
⤷ i'm currently looking at the 1.6.x branch, but it doesn't contain much updates after release
+someone varac_: most of work is going on master for upcoming 2.0 release. for 1.x series updates you should check
1.x.x branch.
⤷ as for security fixes, there is nothing broken so far for urgently fix, though 2.0 received some security
enhancements.

```

**Comment by varac** [comment]
works fine !
